lib/http/chainable.rb

```diff
@@ -62,15 +68,16 @@ def connect(uri, options = {}) # rubocop:disable Style/OptionHash
       request :connect, uri, options
     end
 
-    # Apply partial modifications to a resource
+    # Apply partial modifications to a resource.
+    # See Client#request.
     # @param uri
     # @option options [Hash]
     def patch(uri, options = {}) # rubocop:disable Style/OptionHash
       request :patch, uri, options
     end
 
-    # Make an HTTP request with the given verb
-    # @param (see Client#request)
+    # Make an HTTP request with the given verb.
+    # See Client#request.
```

Comment:
Please use `@see` tag

```suggestion
    # @see Client#request
```